core/layout_api.php

```diff
@@ -230,17 +230,20 @@ function layout_head_css() {
 	if ( config_get_global( 'cdn_enabled' ) == ON ) {
 		html_css_cdn_link( 'https://maxcdn.bootstrapcdn.com/bootstrap/' . BOOTSTRAP_VERSION . '/css/bootstrap.min.css' );
 		html_css_cdn_link( 'https://maxcdn.bootstrapcdn.com/font-awesome/' . FONT_AWESOME_VERSION . '/css/font-awesome.min.css' );
+
+		# theme text fonts
+		html_css_cdn_link( 'https://fonts.googleapis.com/css?family=Open+Sans' );
 	} else {
 		html_css_link( 'bootstrap-' . BOOTSTRAP_VERSION . '.min.css' );
 		html_css_link( 'font-awesome-' . FONT_AWESOME_VERSION . '.min.css' );
+
+		# theme text fonts
+		html_css_link( 'ace-fonts.css' );
```

Comment:
It will never be the same, as Google serves different css for different OSes and browsers. If we add other fonts, we should also provide valid cdn links.
GF css version is supposed to be best for one's OS and browser, but not always, of course.